### PR TITLE
gitignore a1.vcxproj.user

### PR DESCRIPTION
**Pull request by PenguinFlyingHigh**
added a1.vcxproj.user to gitignore
It contains user-specific overrides, particularly debugger settings.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,4 +41,5 @@ cmake_install.cmake
 /visual_studio/tmp/
 /visual_studio/Debug/
 /visual_studio/.vs/
+visual_studio/a1.vcxproj.user
```
Comment (EmceeEscher) [inline .gitignore]: does this need a slash at the front like rest of the `visual_studio` stuff?

**Review by EmceeEscher — APPROVED** [review]
Left one comment, but if it works, then go ahead and merge. I don't have this file in my project so I can't test

**Comment by theochit** [comment]
Can we throw the log files in here while we're at it? I believe the only extensions are .log and .tlog and they're in a temp folder of visual_studio

File: .gitignore (hunk 1 above)
Comment (PenguinFlyingHigh) [reply .gitignore]: Oh whoops yeah. Good call.

**Pull request merged**